code/modules/admin/admin_topic/misc.dm

```diff
@@ -0,0 +1,36 @@
+
+
+
+
+/datum/datum_topic/admins_topic/stickyban
+	keyword= "stickyban"
+	log = FALSE
+
+/datum/datum_topic/admins_topic/stickyban/Run(list/input,var/datum/admin/A)
+	stickyban(input["stickyban"],input)
```

Comment:
Not sure why but this one, cmd_show_exp_panel and toggle_exempt_status didnt work with trying A.